plugins/tag/commands

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+set -eo pipefail; [[ $DOKKU_TRACE ]] && set -x
+source "$PLUGIN_PATH/common/functions"
+source "$(dirname $0)/functions"
+
+case "$1" in
+  tag:add)
+    [[ -z $2 ]] && echo "Please specify an app to run the command on" && exit 1
+    APP="$2"; IMAGE_TAG="$3"; IMAGE_REPO=$(get_app_image_repo $APP)
+    verify_app_name "$APP"
+
+    tag_image "$IMAGE_REPO:latest" "$IMAGE_REPO:$IMAGE_TAG"
+    dokku_log_info2_quiet "Added $IMAGE_TAG tag to $IMAGE_REPO"
+    pluginhook tag-add $APP $IMAGE_TAG
+    ;;
+
+  tag:deploy)
+    [[ -z $2 ]] && echo "Please specify an app to run the command on" && exit 1
+    APP="$2"; IMAGE_TAG="$3"
+    verify_app_name "$APP"
+
+    release_and_deploy $APP $IMAGE_TAG
+    ;;
+
+  tag:list)
+    [[ -z $2 ]] && echo "Please specify an app to run the command on" && exit 1
+    APP="$2"; IMAGE_REPO=$(get_app_image_repo $APP)
+    verify_app_name "$APP"
+
+    dokku_log_info2_quiet "Image tags for IMAGE_REPO"
+    docker images "IMAGE_REPO"
+    ;;
+
+  tag:rm)
```

Comment:
Is there a pattern? I also see ":remove" used twice and destroy once (also unset, but that makes sense in env context). I'm keen to keeping to the 'nix style shorthand.